openedx/core/djangoapps/schedules/management/commands/setup_models_to_send_test_emails.py

```diff
@@ -74,7 +70,6 @@ def handle(self, *args, **options):
         CourseFactory.reset_sequence(max_org_sequence_id + 1, force=True)
         course = CourseFactory.create(
             start=datetime.datetime.today() - datetime.timedelta(days=30),
-            start_date=datetime.datetime.today() - datetime.timedelta(days=30),
```

Comment:
You removed the wrong line here, no?

Reply:
No, this was to be removed from step 4/5 but was left. I noticed it during rebase.

Reply:
I think this change is incorrect - it's still writing to the `start` field even though this PR removes that field.

Reply:
Modulestore Coursefactory is used in multiple places with same `start` args.
https://github.com/edx/edx-platform/blob/61e1eda20df2825a409db3e2d36c69d7c36d3e2d/lms/djangoapps/experiments/tests/test_views_custom.py#L74
https://github.com/edx/edx-platform/blob/61e1eda20df2825a409db3e2d36c69d7c36d3e2d/lms/djangoapps/experiments/tests/test_views_custom.py#L88
https://github.com/edx/edx-platform/blob/a3aa6d358ca3e335183cb24b9eabe2113aaeb30c/openedx/features/course_experience/tests/views/test_course_dates.py#L29

Reply:
So, this is a different `start` than the one from Schedule?

Reply:
Yes.